koku/api/cloud_accounts/views.py

```diff
@@ -15,23 +15,49 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 """View for Cloud Account."""
-from rest_framework import viewsets
-from rest_framework.permissions import AllowAny
+import json
+import logging
+import os
 
-from api.cloud_accounts.models import CloudAccount
-from api.cloud_accounts.serializers import CloudAccountSerializer
+from django.core.paginator import Paginator
+from django.shortcuts import render
+from rest_framework import permissions
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.decorators import permission_classes
+from rest_framework.decorators import renderer_classes
+from rest_framework.renderers import BrowsableAPIRenderer
+from rest_framework.renderers import JSONRenderer
+from rest_framework.response import Response
 
+from koku.settings import STATIC_ROOT
 
-class CloudAccountViewSet(viewsets.ReadOnlyModelViewSet):
-    """View for Cloud Accounts."""
+LOG = logging.getLogger(__name__)
 
-    serializer_class = CloudAccountSerializer
-    permission_classes = (AllowAny,)
+OPENAPI_FILE_NAME = os.path.join(STATIC_ROOT, "cloud_accounts.json")
+"""View for Cloud Accounts."""
 
-    def get_queryset(self):
-        """Override default get_queryset to filter on name."""
-        queryset = CloudAccount.objects.all()
-        cloud_account = self.request.query_params.get("name", None)
-        if cloud_account is not None:
-            queryset = queryset.filter(name=cloud_account)
-        return queryset
+
+def get_json(path):
+    """Obtain API JSON data from file path."""
+    json_data = None
+    with open(path) as json_file:
+        try:
+            json_data = json.load(json_file)
+        except (IOError, json.JSONDecodeError) as exc:
+            LOG.exception(exc)
+    return json_data
+
+
+@api_view(["GET"])
+@permission_classes((permissions.AllowAny,))
+@renderer_classes([BrowsableAPIRenderer, JSONRenderer])
+def cloudaccounts(request):
+    """Provide the openapi information."""
+    data = get_json(OPENAPI_FILE_NAME)
+    paginator = Paginator(data, 25)
+    page_number = request.GET.get("page")
+    page_obj = paginator.get_page(page_number)
+    if data:
+        return Response(list(page_obj))
+    return Response(status=status.HTTP_404_NOT_FOUND)
```

Comment:
Output format should match with the spec provided above. A 404 doesn't make sense as you will return the above spec with an empty *data* field (i.e. `"data": []`)